CONTRIBUTING.md

```diff
@@ -31,8 +31,15 @@ examples.
 
 Pull request
 ------------
-Provide the same information as you would for a [feature request](#feature-request). Also briefly
-describe what the implementation might look like.
+Provide the same information as you would for a [feature request](#feature-request).
+
+We ask that you:
+
+* [Write a good commit message](http://chris.beams.io/posts/git-commit/)
+* Issue 1 Pull Request per feature. Don't lump unrelated changes together.
```

Comment:
Heh, by accident I think you're breaking your own rule here.  I don't think these changes belong in this PR :stuck_out_tongue:.